spec/support/active_record/models.rb

```diff
@@ -13,6 +13,11 @@ def name=(val)
       end
     end
 
+    class UserExceptPassword < ::ActiveRecord::Base
```

Comment:
I included this to get around using the `User` model because of the issues I mentioned with respect to the test suite changing the behavior of things.